examples/windows-vm/example.tf

```diff
@@ -1,5 +1,13 @@
 provider "azurerm" {
   features {}
+  subscription_id = "01111111111110-11-11-11-11"
```

Comment:
Replace it with 
`"000000-11111-1223-XXX-XXXXXXXXXXXX"`

Reply:
its done